py_mpwp_v5_0/log_modified_parameters.txt
Model py_mpwp v5.0
Authors: K. Thielen and L. Biddle 2020
Contributors: F. Roquet
 
Model parameter: kt_switch = 1
Model parameter: bc_ice = 0
Model parameter: full_ice = 1
Model parameter: ocean_relax_switch = 0
Model parameter: ekman_switch = 1
Model parameter: diffusion_switch = 1
Model parameter: rb = 0.65
Model parameter: rg = 0.25
Model parameter: dt = 1200
Model parameter: freq_save = 1.0
Model parameter: init_day = 22
Model parameter: ndays = 225
Model parameter: dz = 3.0
Model parameter: max_depth = 750.0
Model parameter: ml_min = 3.0
Model parameter: lat = 60.0
Model parameter: output_file = output_modified_parameters.nc
Model parameter: log_file = log_modified_parameters.txt
Model parameter: forcing_file = forcing.nc
Model parameter: init_state_file = init_state.nc
Model parameter: equation_of_state = linear
Model parameter: alpha0 = 5.82e-05
Model parameter: beta0 = 0.0008
Model parameter: rho_ocean_ref = 1026.0
Model parameter: kappa = 0.0001
Model parameter: kappa_u = 5e-05
Model parameter: w_ekman = 0.0
Model parameter: OR_days = 180.0
Model parameter: ad_depth = 597.0
Model parameter: A_0 = 0.9
Model parameter: h_i0 = 0.5
Model parameter: Div_yr = 0.5
Model parameter: h_snow = 0.1
 
Maximum depth: 750.0 meters
 
iter,ml_depth,si_th,si_co,sst,sss,ml_index
72.00 15.0 0.45306981184556355 0.8709272041249236 -1.8331821492029405 34.17193261778288 5
144.00 3.0 0.4415046672493294 0.8597134247559995 -1.840629858498025 34.1700939616731 1
216.00 90.0 0.4392548170688516 0.8520443216195438 -1.8448180687748081 34.17163433939017 30
288.00 93.0 0.4453754822422414 0.8520437079234989 -1.8485777107659287 34.17394767067025 31
360.00 96.0 0.4477983806197159 0.8538039509477892 -1.846801772671844 34.1756424978022 32
432.00 96.0 0.45224146523772624 0.8566755514837779 -1.8478196369653612 34.178010053981865 32
504.00 99.0 0.4557990946627907 0.8587070663340372 -1.8469337725408512 34.17989184292697 33
576.00 96.0 0.4588246794966062 0.856451863170225 -1.847680265935303 34.1809453588273 32
648.00 3.0 0.4631498347825442 0.8562702235133458 -1.8458300178803964 34.18290141340273 1
720.00 96.0 0.4629576654909519 0.8510222796194684 -1.8455374332052887 34.182882790035706 32
792.00 3.0 0.4649885518631628 0.8479676157354246 -1.8463516147268668 34.18350595682394 1
864.00 96.0 0.4669959638494689 0.845329055740541 -1.8468088657613397 34.18446950407118 32
936.00 3.0 0.4689933445028674 0.8435274758708762 -1.8472134219839733 34.18553819868976 1
1008.00 96.0 0.47157342767647004 0.8437342982134625 -1.847606662948996 34.18695142204479 32
1080.00 39.0 0.47228986569199194 0.8417750173962448 -1.8462284545881016 34.18723806497416 13
1152.00 96.0 0.47557780806362915 0.8382330279074625 -1.8467107051003144 34.188786404673245 32
1224.00 87.0 0.4762708924700011 0.8336182008910978 -1.8454637300909318 34.189350731531285 29
1296.00 99.0 0.4792645412867249 0.8304361534139231 -1.8464950624606926 34.19077502340139 33
1368.00 102.0 0.48558594325527205 0.833197786051979 -1.848114719523246 34.19416375015847 34
1440.00 99.0 0.4921959132305362 0.8368876931284482 -1.8504207608768242 34.19699394198713 33
1512.00 102.0 0.500764853246255 0.8470469698441476 -1.8494915717916887 34.20151008096399 34
1584.00 102.0 0.5106455407778556 0.8577050604314933 -1.8520348813038867 34.20615793361955 34
1656.00 102.0 0.5194504734800626 0.8694062795042564 -1.8516341627607902 34.21082112326819 34
1728.00 3.0 0.5249236192545402 0.870826624523565 -1.8502262140218968 34.21395627808099 1
1800.00 105.0 0.5353392364663991 0.8758380240712471 -1.852782457107474 34.218187855987566 35
1872.00 105.0 0.5455324384636504 0.8758874237837977 -1.851698921211884 34.2216124312268 35
1944.00 105.0 0.5554174591704286 0.8789844959076809 -1.8515835816177046 34.2254184368444 35
2016.00 105.0 0.5599298662495157 0.8787201330835493 -1.8485874373019604 34.227603233705764 35
2088.00 96.0 0.5609617175058321 0.8739696080741691 -1.847740940501735 34.227706562045576 32
2160.00 3.0 0.5648999445797737 0.8701009428985262 -1.8461803354857815 34.23012227215813 1
2232.00 108.0 0.5698363872221685 0.8661245386913627 -1.8486541959879421 34.23225184175169 36
2304.00 105.0 0.5765384580418028 0.8625125177100531 -1.85041540652827 34.2342125071298 35
2376.00 105.0 0.5831241399380377 0.8628265640503694 -1.850285566530802 34.23696817154222 35
2448.00 105.0 0.589090300836126 0.8612109974292912 -1.8508561815684463 34.239181240522356 35
2520.00 105.0 0.592238155281283 0.8594900038848328 -1.8487399197712777 34.24084993761175 35
2592.00 102.0 0.5941163040112571 0.8547661434095422 -1.8491637836297927 34.24135267989479 34
2664.00 105.0 0.5971367700091046 0.8512723125631191 -1.8486238406671656 34.242963365704554 35
2736.00 3.0 0.5986879586668347 0.8461811440142395 -1.8480685690264453 34.243963207406885 1
2808.00 24.0 0.5949264236896386 0.8400445167113123 -1.8475026550729512 34.24289444315915 8
2880.00 99.0 0.5964234936601541 0.8348365157755605 -1.8492027757236182 34.24381527626499 33
2952.00 102.0 0.6000359333966988 0.8307974151539953 -1.8475146479168103 34.245382878283856 34
3024.00 102.0 0.6068817351576505 0.8304256505708679 -1.8489855601693186 34.24867383398942 34
3096.00 3.0 0.6089752554524528 0.8265467817064054 -1.8487851823116659 34.24973806711318 1
3168.00 3.0 0.6032417172772991 0.8198107963242504 -1.8476770453388562 34.24870461354157 1
3240.00 3.0 0.6032259263871381 0.8145835627937718 -1.8484317592273707 34.24924794807857 1
3312.00 24.0 0.5907706559414238 0.8050323459230094 -1.845808629098515 34.24720546691615 8
3384.00 102.0 0.5874353931996121 0.79808408587861 -1.849502656143676 34.2478919906014 34
3456.00 3.0 0.5901068117198589 0.7987588047903131 -1.8511281589574498 34.24953506929779 1
3528.00 3.0 0.5903351451592975 0.7963499454708122 -1.8495681760204599 34.250173164210985 1
3600.00 24.0 0.5894620957299567 0.7927329894807241 -1.8486096079557344 34.250402277343184 8
3672.00 21.0 0.5860969707419049 0.7863955004500969 -1.847842223097288 34.25064079727911 7
3744.00 3.0 0.5866194594172046 0.7822991214217937 -1.850615470767359 34.25123051629317 1
3816.00 21.0 0.5856947448312018 0.7798995025858666 -1.8485457284748243 34.25180636459467 7
3888.00 21.0 0.5821793904592193 0.7742236743415745 -1.848766370455882 34.2505192799694 7
3960.00 18.0 0.5786715929219949 0.7686431758871999 -1.8485687560780426 34.249309337167844 6
4032.00 30.0 0.5721576267407813 0.7619614859725725 -1.8482468326497847 34.24715083232386 10
4104.00 3.0 0.5401640274670544 0.7429083736983022 -1.8408504028095085 34.247368026335685 1
4176.00 3.0 0.5370106435892235 0.7364653033309378 -1.8507916728884193 34.24900103796359 1
4248.00 111.0 0.538702977000548 0.7358661647002197 -1.849993266004781 34.249873638014606 37
4320.00 117.0 0.543287020310968 0.7431302761051177 -1.850449251451764 34.25374301548734 39
4392.00 117.0 0.5483512956417801 0.7544082467202757 -1.851450499928583 34.257515126287764 39
4464.00 114.0 0.5538977158043761 0.7593703649429597 -1.8531218931951856 34.25994747903724 38
4536.00 3.0 0.5608465042486427 0.7698943945959321 -1.8566335390301414 34.26338068512618 1
4608.00 114.0 0.5663065538038689 0.7813987664340156 -1.8536355012117265 34.26695847432219 38
4680.00 114.0 0.5711245638103192 0.7835845735132246 -1.8522439543083409 34.26920663998765 38
4752.00 114.0 0.5741969644806009 0.7831814330177568 -1.8509811169536614 34.27053914531135 38
4824.00 111.0 0.5782733970587893 0.7797392123541929 -1.8507114825249427 34.27158400672922 37
4896.00 108.0 0.5811264955622286 0.7754354449593421 -1.8508656649767739 34.27217337184638 36
4968.00 108.0 0.5845137475132528 0.7722606876928811 -1.8515623893014335 34.273125962166816 36
5040.00 108.0 0.5843412824975437 0.7680518103451619 -1.85050125663357 34.2735070267732 36
5112.00 108.0 0.5874794056460696 0.7645759304232935 -1.851070620010376 34.274242282528206 36
5184.00 3.0 0.581941262316904 0.7582418493867986 -1.8474880929059805 34.27298315753743 1
5256.00 114.0 0.5795578013716546 0.7514765873132386 -1.8515737036732516 34.274574318350695 38
5328.00 108.0 0.5780992335450068 0.7509894836438172 -1.8504876785431383 34.27730761073712 36
5400.00 3.0 0.5783703358624681 0.7478928574501201 -1.8498533767991823 34.27850419443737 1
5472.00 3.0 0.5771599273674152 0.7427376481942402 -1.851723865490698 34.27845285616899 1
5544.00 15.0 0.576056013618602 0.739287506879067 -1.851092713523595 34.274744578854786 5
5616.00 57.0 0.5759371692759452 0.7364204964013459 -1.853461972000477 34.277291299655424 19
5688.00 108.0 0.5819051440604712 0.7375175035084935 -1.8531341854665304 34.28029587911508 36
5760.00 111.0 0.5880027390207675 0.740160497893984 -1.8526685929006719 34.28278666762192 37
5832.00 3.0 0.5934778552560629 0.7430860603423473 -1.850060814413037 34.2855001960315 1
5904.00 3.0 0.5874912308074648 0.7369678415888664 -1.8527345821541419 34.284325463901034 1
5976.00 3.0 0.5916093727007065 0.7399681045135994 -1.8527107005392973 34.2869502362274 1
6048.00 3.0 0.5954318257171392 0.7399685133285353 -1.8536449730646403 34.288659856492046 1
6120.00 24.0 0.5932282331095481 0.7402212923338599 -1.8499535830621816 34.28878495699726 8
6192.00 3.0 0.5825693264341711 0.7320949833870918 -1.8505823427223411 34.28528422051165 1
6264.00 96.0 0.5803199823131445 0.7288909205990646 -1.8523950361430148 34.28479836678411 32
6336.00 117.0 0.5853552705839281 0.729876367706999 -1.851956373896979 34.288285731848184 39
6408.00 3.0 0.5873130264490158 0.7311421938937341 -1.8504269021535802 34.289932760315665 1
6480.00 3.0 0.5742535180274834 0.7223025910863177 -1.8454444323265977 34.28914018389617 1
6552.00 3.0 0.5679792840603755 0.7140304602960216 -1.8513583992626905 34.28995728415575 1
6624.00 72.0 0.5587765704056892 0.7064259358711588 -1.8512134425871742 34.287703231503436 24
6696.00 96.0 0.5538169654806394 0.7001334714227486 -1.8516479808120603 34.28672596347882 32
6768.00 21.0 0.5499440901194811 0.6952218330129758 -1.850126124621787 34.28148106672058 7
6840.00 102.0 0.5377241831918813 0.6861864876343278 -1.851019491438003 34.281855011050055 34
6912.00 105.0 0.5254622201308762 0.676815532809616 -1.8506019664176547 34.282330874172956 35
6984.00 33.0 0.5224369045745924 0.6719969631349086 -1.8514500903862725 34.27890651725172 11
7056.00 3.0 0.5241415791645051 0.6738854035041301 -1.853222695780509 34.28269288254515 1
7128.00 108.0 0.5244235125629343 0.6734280927297459 -1.850550159921413 34.28430539979571 36
7200.00 21.0 0.513291921446053 0.6647392942109736 -1.8504300189013188 34.279493485534594 7
7272.00 93.0 0.49855330231974054 0.6540711132006358 -1.850500821918842 34.27758345093958 31
7344.00 84.0 0.4905394119171079 0.6466841556560069 -1.850792822488508 34.275007216941056 28
7416.00 3.0 0.48986281014287153 0.6449199753865635 -1.8520152992892331 34.2761459880277 1
7488.00 81.0 0.4829384457591724 0.6399597245884523 -1.850600653328579 34.27551464503963 27
7560.00 75.0 0.47231155146048653 0.6312756812209713 -1.8506075972753853 34.273089331709464 25
7632.00 84.0 0.4624124095757238 0.6234520771932234 -1.850936071581331 34.27046447611132 28
7704.00 12.0 0.457510440456686 0.6198445458948139 -1.8510858216891282 34.26226421973892 4
7776.00 18.0 0.45286858521353607 0.615615831223484 -1.8513149539051903 34.25536591698887 6
7848.00 9.0 0.4453871906595528 0.6112439524669094 -1.8495023733039242 34.25399530265922 3
7920.00 21.0 0.4347603563521164 0.6027018333437435 -1.8492512254185929 34.24582853216597 7
7992.00 33.0 0.4187791405351424 0.5916163595680144 -1.8495207144910413 34.24399803336247 11
8064.00 54.0 0.4052071843430804 0.5841810422446261 -1.849555209839259 34.24268870059189 18
8136.00 57.0 0.4021967772383427 0.5846614143857672 -1.8511171763787502 34.24201889886355 19
8208.00 51.0 0.39409933419851123 0.5855548713582179 -1.8493251000604372 34.24072422073299 17
8280.00 27.0 0.3773805757405294 0.5731384598026468 -1.84830690460054 34.23249865764849 9
8352.00 33.0 0.36537519460478424 0.5626812855253859 -1.8481517668089964 34.22069097385174 11
8424.00 36.0 0.3529269233270946 0.5530256812736357 -1.8490535383666389 34.21567346909217 12
8496.00 21.0 0.33956539997738183 0.5436071605743346 -1.8469034838696308 34.214190385741034 7
8568.00 33.0 0.3215315998580142 0.5278144050999597 -1.8471724292183038 34.206291750054355 11
8640.00 39.0 0.3048714620608539 0.5147057145174374 -1.8466928897170025 34.19699126846241 13
8712.00 6.0 0.29208494848598077 0.5031222253558958 -1.846028786712948 34.18960764144081 2
8784.00 18.0 0.27249981756374286 0.48435731927022296 -1.8453718219248627 34.18052102715454 6
8856.00 51.0 0.2507904394405925 0.4653664446664307 -1.8466507322708985 34.175071471446635 17
8928.00 15.0 0.24044078744639047 0.4595670961685888 -1.8465495452217942 34.15955314808992 5
9000.00 48.0 0.225882630099455 0.45816496413299956 -1.84642580355459 34.1624101568233 16
9072.00 9.0 0.2153177523335064 0.4598553790085908 -1.8424153945754378 34.14793097660607 3
9144.00 30.0 0.19627173500212622 0.4385216722263306 -1.844784942782706 34.14421328129701 10
9216.00 15.0 0.18417552510942414 0.4249434387265516 -1.8432863055791764 34.138522492189566 5
9288.00 45.0 0.15987536485499204 0.3946463330903089 -1.844008619253926 34.13226312074152 15
9360.00 39.0 0.1422996427287936 0.3832858533622062 -1.844462090834962 34.12463798802069 13
9432.00 27.0 0.13034861919476712 0.38835747891310635 -1.8439789964954325 34.11523794587613 9
9504.00 33.0 0.11469707292499436 0.386614458794007 -1.8437467577180018 34.10897589718752 11
9576.00 24.0 0.10258958265713004 0.38947068765690984 -1.8443983689501802 34.09950379313471 8
9648.00 24.0 0.09313948513738073 0.40013165246856236 -1.8433831419522302 34.09297033062219 8
9720.00 3.0 0.08256609403075407 0.3985871519524887 -1.8405158318270112 34.09167937469021 1
9792.00 27.0 0.06595640681064645 0.35973629025291604 -1.8414890277888216 34.08131917293411 9
9864.00 15.0 0.05336342012607812 0.3536876747150671 -1.8416447261028122 34.06445707144491 5
9936.00 18.0 0.03919577154610484 0.3435169875537932 -1.841185275307676 34.0515668215948 6
10008.00 21.0 0.022801261605136316 0.3157658386239974 -1.8390806586121202 34.04770790892758 7
10080.00 12.0 0.0 0.0 -1.801223359703974 34.049716508242916 4
10152.00 15.0 0.0 0.0 -1.7368315387043012 34.04871381564696 5
10224.00 21.0 0.0 0.0 -1.6303322591903984 34.04874920564679 7
10296.00 15.0 0.0 0.0 -1.5438059945087292 34.04819450863447 5
10368.00 30.0 0.0 0.0 -1.4837855998663338 34.052310035625844 10
10440.00 33.0 0.0 0.0 -1.3928338269631775 34.053109242040605 11
10512.00 21.0 0.0 0.0 -1.3224650976265178 34.050113613689525 7
10584.00 18.0 0.0 0.0 -1.234675135111958 34.04956133399592 6
10656.00 18.0 0.0 0.0 -1.1107886543954777 34.04915320403632 6
10728.00 15.0 0.0 0.0 -1.1130709586051262 34.05163250370406 5
10800.00 27.0 0.0 0.0 -1.074408796724955 34.05167443990602 9
10872.00 6.0 0.0 0.0 -0.9966457606387509 34.048585814458896 2
10944.00 21.0 0.0 0.0 -0.9688747859830823 34.04176920685674 7
11016.00 21.0 0.0 0.0 -0.8777225547958175 34.041310867226066 7
11088.00 24.0 0.0 0.0 -0.8264348165369351 34.039119554745504 8
11160.00 30.0 0.0 0.0 -0.8009045925160584 34.04366959043558 10
11232.00 27.0 0.0 0.0 -0.7040437952445792 34.04448318547032 9
11304.00 27.0 0.0 0.0 -0.615134502838212 34.04518022930909 9
11376.00 24.0 0.0 0.0 -0.6699508999531759 34.04933157715555 8
11448.00 30.0 0.0 0.0 -0.5991073593815768 34.050342837733695 10
11520.00 27.0 0.0 0.0 -0.5326974473287397 34.04972741121583 9
11592.00 30.0 0.0 0.0 -0.44971394513163465 34.04894069411787 10
11664.00 30.0 0.0 0.0 -0.3662201302622177 34.050016716136426 10
11736.00 12.0 0.0 0.0 -0.35669356377350847 34.04605832871618 4
11808.00 30.0 0.0 0.0 -0.33874199531496046 34.048160831112796 10
11880.00 27.0 0.0 0.0 -0.29014816101734514 34.048462753880564 9
11952.00 9.0 0.0 0.0 -0.2669312842155129 34.04849511657952 3
12024.00 27.0 0.0 0.0 -0.22687766771425993 34.046175483124394 9
12096.00 3.0 0.0 0.0 -0.15459948702628584 34.04535054806157 1
12168.00 27.0 0.0 0.0 -0.06147043309960268 34.04525433645647 9
12240.00 30.0 0.0 0.0 -0.0066383550438846774 34.045401483770426 10
12312.00 30.0 0.0 0.0 0.03581439235226828 34.046072414563405 10
12384.00 33.0 0.0 0.0 0.1090137041443351 34.04670500934618 11
12456.00 27.0 0.0 0.0 0.18909788168708566 34.04659557511906 9
12528.00 24.0 0.0 0.0 0.268938961711604 34.04669142011139 8
12600.00 27.0 0.0 0.0 0.3606236636743347 34.04728006137599 9
12672.00 21.0 0.0 0.0 0.4100409365490448 34.048215646011585 7
12744.00 18.0 0.0 0.0 0.4381139608978154 34.04596170714575 6
12816.00 24.0 0.0 0.0 0.546294219770188 34.045607809883286 8
12888.00 27.0 0.0 0.0 0.5773886881264935 34.048176978311126 9
12960.00 30.0 0.0 0.0 0.583228081489323 34.04823181153874 10
13032.00 30.0 0.0 0.0 0.564285215364108 34.04718963950019 10
13104.00 33.0 0.0 0.0 0.6251736071072129 34.0481075786462 11
13176.00 27.0 0.0 0.0 0.6886739033962942 34.048387283239556 9
13248.00 33.0 0.0 0.0 0.7500441106116674 34.04853516866829 11
13320.00 30.0 0.0 0.0 0.7921832711355318 34.0495599181403 10
13392.00 21.0 0.0 0.0 0.8143341455850787 34.04915483633492 7
13464.00 33.0 0.0 0.0 0.7927815407039648 34.050851429400176 11
13536.00 27.0 0.0 0.0 0.7306639631753614 34.05464227926387 9
13608.00 18.0 0.0 0.0 0.742883349563319 34.04480787875726 6
13680.00 21.0 0.0 0.0 0.807897439613238 34.043438156141754 7
13752.00 27.0 0.0 0.0 0.8804963975721372 34.043230153638085 9
13824.00 27.0 0.0 0.0 0.9199514941653748 34.043906476492204 9
13896.00 18.0 0.0 0.0 0.9815197640247997 34.03936239195272 6
13968.00 12.0 0.0 0.0 1.0387042558161543 34.0400094828506 4
14040.00 18.0 0.0 0.0 1.1277405945696304 34.03755822967141 6
14112.00 12.0 0.0 0.0 1.225030138535512 34.036740140676706 4
14184.00 24.0 0.0 0.0 1.2537160721982514 34.038283576702746 8
14256.00 24.0 0.0 0.0 1.2519739079556729 34.04044045533009 8
14328.00 21.0 0.0 0.0 1.2548374348715163 34.02146669290917 7
14400.00 30.0 0.0 0.0 1.1674649383819238 34.03412184059271 10
14472.00 33.0 0.0 0.0 1.1952469673637318 34.035533452616725 11
14544.00 18.0 0.0 0.0 1.2268371768368083 34.034273619017775 6
14616.00 27.0 0.0 0.0 1.2590914312143062 34.03572433314064 9
14688.00 33.0 0.0 0.0 1.2794934520601262 34.03676960700658 11
14760.00 30.0 0.0 0.0 1.268686769655861 34.039277781109256 10
14832.00 27.0 0.0 0.0 1.306934136885174 34.039275502900985 9
14904.00 30.0 0.0 0.0 1.3317858354863064 34.04023888746549 10
14976.00 30.0 0.0 0.0 1.35864735451446 34.040567374214326 10
15048.00 27.0 0.0 0.0 1.3916491366789387 34.04172446359804 9
15120.00 27.0 0.0 0.0 1.3529928435716074 34.0361294456343 9
15192.00 30.0 0.0 0.0 1.3821432061863035 34.03642768320185 10
15264.00 30.0 0.0 0.0 1.4142608624891422 34.034764954505576 10
15336.00 30.0 0.0 0.0 1.4429683856717563 34.03570720756614 10
15408.00 30.0 0.0 0.0 1.4635923137160767 34.036510516720384 10
15480.00 27.0 0.0 0.0 1.4809223467684431 34.037730955613725 9
15552.00 24.0 0.0 0.0 1.5065604867290447 34.037620719527474 8
15624.00 27.0 0.0 0.0 1.5159410053527191 34.038054214668186 9
15696.00 27.0 0.0 0.0 1.527915166523775 34.0389175734861 9
15768.00 18.0 0.0 0.0 1.561831483460676 34.039500812851394 6
15840.00 24.0 0.0 0.0 1.5249242042462021 34.04023753469781 8
15912.00 18.0 0.0 0.0 1.5268363445824915 34.03328019757003 6
15984.00 24.0 0.0 0.0 1.5451428536019287 34.03497240719071 8
16056.00 24.0 0.0 0.0 1.466093015485928 34.036355887098075 8
16128.00 27.0 0.0 0.0 1.4683969947728635 34.037103091267916 9
16200.00 27.0 0.0 0.0 1.4602749813170155 34.037056912276796 9


PyMice_PWP_v0.1/changes.txt
Added a few things:
Command line arguements
Sea ice salinity 


PYMice_PWP_v02/params.py
import numpy as np
#todo add ml_depth as param
#todo run with older settings and then turn on rb rg and see if still ok.
###################################################
##          Switches                ##
###################################################
# krauss turner
kt_switch       = 1
#Biddle Clark Ice (from thesis)
bc_ice          = 0
# modified akin to petty et al
full_ice          = 1
#f or conductivity term in heat balance of full_ice- implemented mainly for testing
cond_on     =  1        #
# critical bulk richardson number (0.65) set to 0 to turn off.
rb 		        = 0.65
# critical gradient richardson number (0.25) set to 0 to turn off.
rg		        = 0.25
# this is for inertial internal wave dissipation stuff, all other scripts I've seen have this unlabeled so assume 0
ucon            = 0
# relaxation switch
ocean_relax_switch  =     0


# depth below which "advection" occurs (m), if using this make sure ad_i is passed to
# relaxation function in main
ad		             = 597.


diffusion_switch    = 1
change=                   1
temp_diff	= 1e-4*change 	# diffusion paramater for t and s
salt_diff   = temp_diff
oxy_diff	=temp_diff      # diffusion paramater for DO
vel_diff	= 5e-4*change 	# diffusion of velocity fields.


##############################################
##             Sea Ice Settings             ##
##############################################

# upper limit of sea ice concentration.
A_grow      =                                                                                                       0.9 
# lower limit of sea ice concentration.
A_melt      =                                                                                                      0.2 
# initial sea ice concentration
A_0		    = A_melt
A_min       = 0
#starting ice thickness
h_i0        = 0.
# average bulk salinity of sea ice
S_ice		= 5.
# minimum ice thickness
h_ice_min	= 0.
# snow thickness
h_snow      =       0.0
# divergence of ice concentration (v3 only)
Div_yr =                 -0.8 
# lateral vs basal melting paramater (1 for all basal )
R_b=                 0.6

###################################################
##          Simulation Paramaters               ##
###################################################
dt		    =  100
# time-step increment for saving to file (units seconds/dt)
dt_save		= round(160000./dt)
dz		    =  3.0
# the number of days to run
days 		= 1000.
# the depth to run
depth 		= 750.
# this is a bit deprecated as I've put a line to find the mixed layer of the initial profile in the main
ml_depth_0  = 3.
# minimum allowable mixed layer
ml_min      = 3.
# max ml
ml_max      = depth-dz
# latitude (degrees)
lat 		= 57.

maxiter     = days*8.64e4/dt
nz          = int(depth/dz)+1
z           = np.arange(0,int(nz))*dz
# time scale of relaxation

OR_days     = 0.75

#OR_days_dw          = 11.5      #this is timescale of deepwater fom louise thesis

#convert the relaxation timescale from years to seconds
OR_d               = (OR_days*365.*24.*60.*60.)/dt
OR_timescale              =1./OR_d



#convert ice divergence to seconds
dy=Div_yr
if dy==0:
    Div=0.
    print('rightyo')
else:
    Div=1./(Div_yr*365.*24.*60.*60./dt)

#if relaxation below a specific depth, ad, get the index of this depth

ad_i		= int(ad/dz)
f 		    = 2*7.29*10**(-5)*np.sin(lat*np.pi/180.)
ang 	    = -f*dt/2.	# angle for current rotation. angle equal to inertial rotation for


###################################################
##                  Constants                    ##
###################################################

alpha		    = 5.82*10**(-5)	# thermal expansion coef K^(-1)
beta		    = 8*10**(-4)	# saline contraction coef
cp_air 		    = 1005.		# Specific heat cap of air (J kg^(-1) K^(-1))
cp_ocean	    = 4190.		#

snow_emiss	    = 0.99		#
snow_albedo	    = 0.8		#

ice_emiss       = 0.99      # EBERT
ice_albedo      = 0.63      # bitz1999


#use snow albedo if there is snow, otherwise use ice props.
if h_snow>0.001:
    si_emiss=snow_emiss
    si_albedo=snow_albedo
else:
    si_emiss=ice_emiss
    si_albedo=ice_albedo

ocean_emiss	    = 0.97		#
ocean_albedo	= 0.06		#

g	 	        = 9.81		# gravitational acceleration
if cond_on == 0:
    cond_ice=0.
else:
    cond_ice	    = 2.04		# cond of ice
cond_snow	    = 0.31		# conductivity value for snow
Latent_sub	    = 2.834*10**6	#latent heat of sublimation
Latent_vapor	= 2.501*10**6	# vaporisation
Latent_fusion	= 3.340*10**5	#
P_atm		    = 101325.		# atmospheric pressure (kPa)
rho_ocean_ref	= 1026.		# ocean reference
rho_ice_ref	    = 930.		# ice density
rho_air_ref	    = 1.275		# air density
stef_boltz	    = 5.67*10**(-8)	#stefan boltzmann constant
I_0		        = 0.45		# fraction of shortwave that penetrates open water surface layer Niiler and Kraus 1977
beta1 		    = 0.6		# longwave extinction coefficient (0.6 m)
beta2		    = 20. 		# shortwave extinction coefficient (20 m)
ep		        = 0.62197  	# epsilon, ratio of molecular weight of water and dry air (0.622)
c1		        = 0.8		# Tang 1991 max magnitude of wind stirring in ml
C_turb_i	    = 0.0013	# Ebert and Curry 1993 turbulent exchange over ice
C_turb_o	    = 0.001		# Ebert and Curry 1993 turbulent exchange over leads
Stanton		    = 0.006		# McPhee 1992 mixed layer to sea ice heat transfer
c_unsteady	    = 0.03		# Kim 1976
dw		        = 10.		# Lemke and Manley 1984 scale depth of dissipation
cm              = 0.03
m_kt		= 0.4		# Co-efficient for power provided by wind, in Kraus-Turner
n_kt		= 0.18		# Co-efficient for power provided by buoyancy, in K-T
cd_ocean	= 0.001		# drag coefficient ocean
cd_ice		= 0.0013	# drag coefficient of ice
rkz		    = 0.0001        # background diffusion (0)

